src/index.scss

```diff
@@ -7,6 +7,19 @@
 @tailwind components;
 @tailwind utilities;
 
+@layer base {
```

Comment:
I think so
but since I did not apply tailwind styles on all pages, I kept it cautiously to avoid potential problem on the other pages. After testing more pages we can remove it from _primitive.scss